_pages/dev/components/banner-component.md

```diff
@@ -2,62 +2,62 @@
 title: Banner Component
 ---
 
-The purpose of the Banner component is to render a media in a given page slot. The media can be optimized for the given slot size. The banner is typically used to link to another internal or external page.
+The banner component is used to render a media item in a given page slot. The media item can be optimized for the given slot size. The banner is typically used to link to another internal or external page.
 
 ## CMS Component Binding
 
-There are multipe banner component types in the CMS:
+There are multiple banner component types in the CMS, as follows:
 
--   `BannerComponent`
--   `SimpleBannerComponent`
--   `SimpleResponsiveBannerComponent`
+- `BannerComponent`
+- `SimpleBannerComponent`
+- `SimpleResponsiveBannerComponent`
 
-There is also a `RotatingImagesComponent` which renders a one or multiple `BannerComponents` in a carousel.
+There is also a `RotatingImagesComponent`, which renders one or multiple instances of `BannerComponent` in a carousel.
 
-The different types have been created for historical reasons, there's not much of a point to distinquish them anymore; there's only component implementation in Spartacus for banners. Sparatacus always renders banners using adaptive and responsive features to render the most optimal image for the given device.
+The different banner component types have been created for historical reasons, but there is not much to distinguish them anymore: in Spartacus, there is only component implementation for banners. Spartacus always renders banners using adaptive and responsive features to render the optimal image for the given device.
 
-The CMS Banner component is mapped to the Spartacus implementation as follows:
+The CMS banner component is mapped to the Spartacus implementation as follows:
 
 ```typescript
 <CmsConfig>{
-    SimpleResponsiveBannerComponent: {
-        component: BannerComponent
-    },
-    BannerComponent: {
-        component: BannerComponent
-    },
-    SimpleBannerComponent: {
-        component: BannerComponent
-    }
+  SimpleResponsiveBannerComponent: {
+    component: BannerComponent,
+  },
+  BannerComponent: {
+    component: BannerComponent,
+  },
+  SimpleBannerComponent: {
+    component: BannerComponent,
+  },
 };
 ```
 
-You can use this component mapping to configure an alternative banner implementation. You might need to map all CMS banner component types in case multiple types are used in your project.
+You can use this component mapping to configure an alternative banner implementation. For example, you might need to map all CMS banner component types if multiple types are used in your project.
```

Comment:
> For example, you might need to map all CMS banner component types if multiple types are used in your project.

If you introduce an alternative component, you must map all the CMS banner component types to your new component implementation, similar to the configuration above.

Reply:
👍